ocaml/xcp-rrdd/scripts/rrdd/rrdd.py

```diff
@@ -246,11 +246,11 @@ def set_datasource(self, name, value, description="", units="",
 
     def get_header(self):
         """Get the 'static' first line of the expected output format."""
-        return self.header
+        return self.header  # pytype: disable=attribute-error
```

Comment:
A few codes show that the attribute initialization can be quite challenging, such as the one below, which results `attribute-error` warning with class A, but the script works well. 
```
root@2a613cf7ba14:/# cat test.py
class A:
    def __init__(self):
        self.x = self.y

class B:
    def __init__(self):
        self.x = 1
        self.y = 2

class C(B,A):
    def __init__(self):
        B.__init__(self)
        A.__init__(self)

c=C()
print(c.x)
print(c.y)

root@2a613cf7ba14:/# python3 test.py
2
2
```

Reply:
In this case, Mark is right, we could just do that, because there is no complex subclassing involved like in acefei's example.

However, the bugfix may just use the smallest and safest change possible and a later clean-up can do the conversion to initialize in __init__() and change the check for the attribute like so:
```py
  def __init__():
+     self.dest = ""
```
and later in a method, we've to replace this:
```py
-     if hasattr(self, 'dest'): return
+     if self.dest:  # lazy_init already done, nothing to do on a 2nd call to it:
+         return
```

Caveat: I do not know if OCaml has any method of catching `AttributeError` exceptions and possibly relying on that, but I think that the toolstack developers know.

In this case, it would however not make sense for `xcp-rrdd` to call `__init__()`, then try to call one of the getter functions that return these lazy created attributes, expect this to cause an Exception only to then call the lazy_init function which then creates these attributes.

But then, I know the Python bindings and how `xcp-rrdd` uses this class.

I think it depends on @MarkSymsCtx or @psafont to know and tell if it's safe.

@psafont: It is ready for review/approve,merge as is, or I can push the update the change above.